python/triton/testing.py

```diff
@@ -142,8 +142,19 @@ def do_bench(fn, warmup=25, rep=100, grad_to_none=None, quantiles=None, return_m
 
     di = runtime.driver.active.get_device_interface()
 
+    synchronize_with_timing = di.synchronize
+    try:
+        from torch._dynamo.utils import CHROMIUM_EVENT_LOG, chromium_event_timed
```

Comment:
I do not think we should rely on anything from dynamo

Reply:
If the goal is to inspect triton's autotuner, you can always customize the do_bench function

Reply:
`autotune(do_bench=...)`

Reply:
Gotcha, yeah good point. Thanks!